README.md

```diff
@@ -1,48 +1,17 @@
-# Abstract
+# Hazelcast Member Tool Prototype
```

Comment:
1. Why do you call it "Prototype"? Won't it be really shipped/released?
2. Is it "Hazelcast Member Tool" and not "Hazelcast Command-line Interface"?